src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -122,7 +122,10 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     executionResult match {
       case SuccessfulExecution(outputs, returnCode) => context.parent ! WorkflowActor.CallCompleted(key, outputs, returnCode)
       case AbortedExecution => context.parent ! WorkflowActor.AbortComplete(key)
-      case FailedExecution(e, returnCode) => context.parent ! WorkflowActor.CallFailed(key, returnCode, e.getMessage)
+      case FailedExecution(e, returnCode) =>
+        logger.error("Failing call: " + e.getMessage, e)
+        log.error(e, e.getMessage)
```

Comment:
I defaulted to console mode for the server because on many occasions I was testing Cromwell using the server and it was just more convenient to see the output on standard output rather than `tail -f` the log file in a separate terminal.  I also feel like it provides a better default for newcomers who are experimenting with Cromwell's server and just want to test some things out